### ISSUE TITLE
Uncaught TypeError: Cannot read property 'classList' of null

### ISSUE THREAD

**Issue by avivmu** [mdn/interactive-examples#1633]
https://developer.mozilla.org/en-US/docs/Web/CSS/inline-size

When clicking on the gap **between** two examples I get this shown in console:
```javascript
Uncaught TypeError: Cannot read property 'classList' of null
    at Object.findParentChoiceElem (editor-css.js?v=f6294ac:1)
    at HTMLElement.<anonymous> (editor-css.js?v=f6294ac:1)
```
I marked some part where it happens for easier understanding
![image](https://user-images.githubusercontent.com/19804341/88888255-a8d07980-d246-11ea-9cab-e8523c66ceb3.png)

**Comment by JavascriptLearner815** [comment]
:clap: for the creator of this issue for pointing this out, I'd've never known! 😄

**Comment by tannerdolby** [comment]
I reckon this can be closed with [#587](https://github.com/mdn/bob/pull/587) landing in `mdn/bob` @schalkneethling !

**Comment by JavascriptLearner815** [comment]
This is still open?

**Comment by tannerdolby** [comment]
> This is still open?

Unfortunately, it looks like this issue was fixed with #587 being merged!

This is ready to be closed! @schalkneethling
